### PR TITLE
Fix kv cache

### PR DESCRIPTION
**Pull request by sunghyuckhong**
## 문제상황
- [MCP PR #367](https://github.com/furiosa-ai/model-compressor-private/pull/367) 연동 PR

## 목적(해결방향)
- prefill mode 시에 KV_cache에 [None, None] * num_layer이 아닌 [torch.zeros(0), torch.zeros(0)] * num_layer 를 넣음 

## 구현 설명 Abstract
- 

## 구현 설명 Detail (ex. 추가된 argument)
- GPT-J에서 두 KV_cache form에 대하여 동치 token이 나오는 것을 eval_accuracy_ci를 통해 확인 

## test 통과 내역 (CI 통과내역과 어떤 machine (GPU pod or NPU machien or local)에서 테스트했는지)
- [ ] local machine with 3090
- [x] GPU pod
- [ ] warboy pod
  - `python model-compressor-private/test/ci_test.py -t runtime`
  - `python model-compressor-private/test/ci_test.py -t accuracy`

## TODO (ex. 후속PR예고)
-"python main.py --scenario Offline --model-path ./model/ --dataset-path ./data/cnn_eval_accuracy_ci.json --model_script_path ./quantization/model_script/Qlevel4_RGDA0-W8A8KV8-PTQ.yaml --gpu --accuracy --use_mcp --calib-dataset-path ./data/cnn_dailymail_calibration.json --recalibrate"

### CODE REVIEW

**Review by ByungkeonKo — APPROVED** [review]
LGTM!
